app/controllers/admin_applications_controller.rb

```diff
@@ -0,0 +1,17 @@
+class AdminApplicationsController < ApplicationController
```

Comment:
Nice working breaking the Admin functionality out into a separate controller. 

Another conventional option for naming is putting an ApplicationsController in a directory called `admin` under `controllers`. That way, you have one regular ApplicationsController and one under the admin folder. Then, in this file, your class name would be `Admin::ApplicationsController` to indicate it lives inside a folder called Admin